server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/MessageProperties.java

```diff
@@ -184,7 +211,25 @@ private static boolean outputProperty(MessageProperty p) {
             }
         }
     }
-    
+
+    public enum ReadLevel {
```

Comment:
ReadProfile would be more appropriate

Reply:
Thanks. See https://github.com/linagora/james-project/pull/2923